app/views/layouts/application.html.haml

```diff
@@ -66,10 +66,15 @@
               %div.govuk-grid-column-one-third
                 %h2.govuk-heading-m
                   = t('footer.for_job_seekers')
-                %ul.govuk-footer__inline-list
-                  %li.govuk-footer__inline-list-item
+                %ul.govuk-footer__inline-list.hide-bullet-points
+                  %li.govuk-footer-list-item
                     =link_to t('footer.search_teaching_vacancies'), root_url, class: 'govuk-footer__link'
-
+                  - if authenticated? == false
```

Comment:
```ruby
- unless authenticated?
```

Come to think of it, now that we have two kinds of users, should we at some point look into refactoring the existing auth helpers to be more in line with the Devise jobseeker ones? e.g. `#authenticated?` -> `#hiring_staff_signed_in?` @DFE-Digital/teaching-vacancies